### PR TITLE
fix(amplify-category-auth): get env specific data in externalAuthEnable

### PR DESCRIPTION
**Pull request by yuth**
With muti enviornment category data is stored in both parameter.json and team-provider-info.json. The externalAuthEnable did not retrive data from team-provider-info.json as it was directly reading the data from the disk. Update the method to use util method to load the resource data 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-category-auth/provider-utils/awscloudformation/assets/cognito-defaults.js [hunk 1]
```diff
@@ -99,7 +99,9 @@ const entityKeys = {
 };
 
 const getAllDefaults = (name) => {
-  const projectName = name.projectConfig ? `${name.projectConfig.projectName.toLowerCase()}${sharedId}` : name;
+  const disallowedChars = /[^A-Za-z0-9_]+/g;
```
Comment (kaustavghosh06) [inline packages/amplify-category-auth/provider-utils/awscloudformation/assets/cognito-defaults.js]: This doesn't belong to this PR.

Comment (yuth) [reply packages/amplify-category-auth/provider-utils/awscloudformation/assets/cognito-defaults.js]: This change is gone now after merging with upstream/multienv
